Show OAI sets in settings form. refs #8239

mcantelon opened the pull request:
In the OAI settings form, show available additional OAI sets if additional
sets are enabled.

mcantelon committed:
LDAP authentication support. refs #9765

mcantelon committed:
Show OAI sets in settings form. refs #8239

In the OAI settings form, show available additional OAI sets if additional
sets are enabled.

jraddaoui commented on lib/form/SettingsOaiRepositoryForm.class.php:
```diff
@@ -41,7 +41,7 @@ public function configure()
       'sample_oai_identifier' => new sfWidgetFormInput(array(), array('class'=>'disabled', 'disabled'=>true)),
       'resumption_token_limit' => new sfWidgetFormInput,
-      'oai_additional_sets_enabled' => new sfWidgetFormSelectRadio(array('choices'=>array(1=>'yes', 0=>'no')), array('class'=>'radio'))
+      'oai_additional_sets_enabled' => new arWidgetFormSelectRadioOaiAdditionalSetEnable(array('choices'=>array(1=>'yes', 0=>'no')), array('class'=>'radio'))
```
We usually add spaces between the array arrows. It'd be nice if you could fix that in this widgets block.

mcantelon replied:
Sounds good!

mcantelon committed:
Made accession identifiers editable. refs #9897

Add the ability to create/edit accession identifiers. Add custom validator to prevent
accession identifiers already in use from being entered.

mcantelon committed:
Allow disabling of accession mask. refs #9915

Added a global setting for enabling/disabling the accession mask (enabled by
default).

mcantelon committed:
Adding whitespace to array definitions.

jraddaoui committed:
Finding aid addons, refs: #9627 #9655 #9699 #9700

- Add option to delete or upload a finding aid using the AtoM worker.
- Check ACL update permission to generate, upload and delete.
- Changes in finding aid work to allow the different workflows.
- Add finding aid transcript and status to a DB property and to the ES index.
- Changes in the finding aid sidebar section.
- Rename finding aid job and generate action.
- Add partial update function for ES documents.
- Add missing translations in jobs.
- Show finding aid option from top-level in descendants.
- Add link in information object index page content.
- Allow download and delete after finding aid format changes.
- Add filter and boolean option in adv. search panel.
- Add button for FA download and use PDF icon.
- Small fixes and cosmetic changes.

mcantelon committed:
Added accession identifier validation. refs #9917

Added AJAX and server-side validation of accession identifiers, ensuring each
is unique.

sevein commented:
Merged in `qa/2.4.x`: 7be6b16f3b73347da4ebad48b98a2c36712a7c15.
